Source/ACE/Entity/GenericObject.cs

```diff
@@ -1,10 +1,17 @@
-﻿namespace ACE.Entity
+using ACE.Entity.Actions;
+using ACE.Factories;
+using System;
+using System.Collections.Generic;
+
+namespace ACE.Entity
 {
     public class GenericObject : WorldObject
     {
         public GenericObject(AceObject aceObject)
             : base(aceObject)
         {
+            if (AnimationFrame == null)
+                AnimationFrame = 101;
```

Comment:
What does this mean?  Can you put in a constant, or enum value instead, to make this clear?

Reply:
Need to apply some researched data to this, will fix shortly.